infrastructure/docs/docker/docs/docker-aws.md

```diff
@@ -0,0 +1,52 @@
+#### Deploy to AWS (Building Docker image on AWS)
+
+AWS has the possibility of directly building a Docker image, allowing us to just push our code (with a [valid Dockerfile](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_image.html#create_deploy_docker_image_dockerfile)). 
+
+1. Download the [Elastic Beanstalk Client](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html)
+    ```bash
+        sudo pip install awswebcli
+    ```
+
+2. Configure your access credentials, creating a file in your root folder `~/.aws/credentials`. Make sure it has the following format:
+    ```bash
+        [profile_name]
+        aws_access_key_id = your_access_key
+        aws_secret_access_key = your_access_key_secret
+    ```
+
+3. Create a Dockerrun.aws.json file in the root directory of your application. It should look like this:
```

Comment:
(This link)[https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_image.html#create_deploy_docker_image_dockerrun] provides some more info

Reply:
Great! I'm going to add this link